app/Http/Controllers/Api/v1/EntityApiController.php

```diff
@@ -81,8 +81,12 @@ public function patch(PatchEntity $request, Campaign $campaign, Entity $entity)
     {
         $this->authorize('access', $campaign);
         $this->authorize('update', $entity);
-
-        $entity->update($request->only('name', 'type', 'is_private', 'is_template', 'tooltip', 'entry', 'image_uuid', 'header_uuid'));
+        if ($entity->hasChild()) {
```

Comment:
Please refactor to remove the copy-paste content, otherwise the day we add a field, we 100% will forget to add it in both places